subiquitycore/log.py

```diff
@@ -13,12 +13,16 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+import grp
 import logging
 import os
 
 
 def setup_logger(dir, base='subiquity'):
-    os.makedirs(dir, exist_ok=True)
+    os.makedirs(os.path.basename(dir), exist_ok=True)
```

Comment:
Why this change? It seems not needed for the permission and probably wrong (or it relies on the helper to run in the right location?)

Reply:
Because I shouldn't write code late at night :) (I had a version that passed a mode to mkdir but that ran into issues with umask)